name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/TradeRepublicPDFExtractor.java

```diff
@@ -1037,7 +1040,124 @@ private void addDeliveryInOutBoundTransaction()
+                                + ") .* ([\\.,\\d]+ Stk\\.)$")
+                .assign((t, v) -> {
+                    if ("Einbuchung".equals(v.get("type")))
+                    {
+                        t.setType(PortfolioTransaction.Type.DELIVERY_INBOUND);
+                        t.setNote("Einstandskurs fehlt");
+                    }
+                })
 
+                // @formatter:off
+                // 1 Ausbuchung AMUN.GOV.BD EO BR.IG 1-3 U.ETF 50 Stk.
+                // Actions au Porteur o.N.
+                // FR0010754135
+                //
+                // 1 Einbuchung MUL-LYX.EO Gov.Bd 1-3Y(DR)U.E. 67,541 Stk.
+                // Nam.-An. Acc o.N.
+                // LU1650487413
+                // @formatter:on
+                .section("position", "name", "shares", "nameContinued", "isin")
+                .match("^(?<position>[\\d]) (Einbuchung|Ausbuchung) (?<name>.*) (?<shares>[\\.,\\d]+) Stk\\.$")
+                .match("^(?<nameContinued>.*)$")
+                .match("^(?<isin>[A-Z]{2}[A-Z0-9]{9}[0-9])$")
+                .assign((t, v) -> {
+                    Map<String, String> context = type.getCurrentContext();
+                    context.put("position", v.get("position"));
+                    t.setDateTime(asDate(context.get("date")));
+                    t.setShares(asShares(v.get("shares")));
+                    t.setSecurity(getOrCreateSecurity(v));
+                    t.setCurrencyCode(asCurrencyCode(t.getSecurity().getCurrencyCode()));
+                    t.setAmount(0L);
+
+                    context.put("name", v.get("name"));
+                    context.put("nameContinued", v.get("nameContinued"));
+                    context.put("isin", v.get("isin"));
+                    context.put("shares", v.get("shares"));
+                })
+
+                .wrap(t -> {
+                    return new TransactionItem(t);
+                });
+        
+        Transaction<AccountTransaction> pdfTransaction2 = new Transaction<>();
+
+        firstRelevantLine = new Block("^ABRECHNUNG$");
+        type.addBlock(firstRelevantLine);
+        firstRelevantLine.set(pdfTransaction2);
+
+        pdfTransaction2 //
+
+                        .subject(() -> {
+                            AccountTransaction accountTransaction = new AccountTransaction();
+                            accountTransaction.setType(AccountTransaction.Type.TAXES);
+                            return accountTransaction;
+                        })
+                                                
+                        // @formatter:off
+                        // 1 Kapitalertragsteuer -21,87 EUR
+                        // Solidaritätszuschlag -1,20 EUR
+                        // GESAMT -23,07 EUR
+                        // @formatter:on
+                        .section("position", "amount", "currency", "date") //
+                        .match("^(?<position>[\\d]) Kapitalertragsteuer \\-[\\.,\\d]+ ([\\w]{3})$") //
```

Comment:
Is this variable "position" reused?
```suggestion
                        .section("amount", "currency", "date") //
                        .match("^[\\d] Kapitalertragsteuer \\-[\\.,\\d]+ ([\\w]{3})$") //
```

Reply:
oops. added the comparison from in/outbound delivery with tax booking